modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectFieldLocalServiceTest.java

```diff
@@ -186,7 +186,7 @@ public void testAddCustomObjectField() throws Exception {
 
 		// Localized object fields must not be required
 
-		_assertFailure(
```

Comment:
Thank you for fixing the test. 
I see that you added one commit to fix this. Because of that, I would like to give feedback regarding this. As a reviewer, it is better to review all the changes in the original commit that introduced this change. In this case, you create a new test and after you fix this test, so this fixing should be squashed in the original commit. 

Gathering all the modifications in the original commit makes the review easier to understand.

A similar situation happened in this PR:
- [LPS-184537 Localizable object fields can not be required](https://github.com/liferay-objects/liferay-portal/pull/2174/commits/e333fe89d68921533536408ae8bc03596c42646e)
- [LPS-184537 Sort](https://github.com/liferay-objects/liferay-portal/pull/2174/commits/bf3b834d5fea9a9919802d31540af6c8fdfe022b)

When creating PR, it is ideal to use the commits to create a story telling about your work. So, splitting the work if it is related is not ideal. Doing this can create confusion for the reviewer.

Do you agree? Let me know if you want to talk about it.
Could you keep in mind and try to apply this in the next PR? :smile:

Reply:
In some cases, adding a new commit makes sense to me because I'm making sure I haven't changed other commits. I thought this might help the reviewer because he would only have to look at the commit that was modified, as the others were already seen.

Your comment also makes sense to me, I will keep it in mind and apply in the next PR :smile:

Reply:
Thank you!! :hugs:

Reply:
> In some cases, adding a new commit makes sense to me because I'm making sure I haven't changed other commits.
If it is related, it is better to be together because I could indicate this change, and, in the end, the comment was discarded because you edited it in future commits. See the "final picture" is good for the reviewer.